source/acquaman/BioXAS/BioXASXASScanConfiguration.cpp

```diff
@@ -142,6 +142,21 @@ AMScanAxisEXAFSRegion* BioXASXASScanConfiguration::createEXAFSRegionInKSpace(dou
 	return region;
 }
 
+QString BioXASXASScanConfiguration::regionsOfInterestHeaderString(const QList<AMRegionOfInterest *> &regions) const
```

Comment:
Could this method be promoted to AMGenericStepScanConfiguration? It might be a nice convenience, if other scan configurations would represent the regions the same way.